conftest.py

```diff
@@ -573,7 +573,7 @@ def run_timer(settings):
                 timeout -= 5
                 parent_status = get_parent_status()
             if parent_status != "already dead":
-                logger.info(f"Timing out test case")
+                logger.warning(f"This test has hung.. Timing out test case")
```

Comment:
Not blocking but maybe put an ellipses `...` lolololol

Reply:
Or more clearly: `This test seems to have hung...`

Reply:
changed lololol